### PR TITLE
Update bootstrap.md

### PR DESCRIPTION
**Pull request by LionelLindt**

### CODE REVIEW

File: ruby_on_rails/bootstrap.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Add them with:
 and configure them in `environment.js`:
 
 ```js
+const webpack = require('webpack')
```
Comment (coorasse) [inline ruby_on_rails/bootstrap.md]: why did you add this? a comment or description would halp me to review the PR.

Comment (LionelLindt) [reply ruby_on_rails/bootstrap.md]: Doesn't the variable `webpack` that is used [here](https://github.com/renuo/applications-setup-guide/blob/22dea33199f9aa77e7f45a04fc437ac0f7aacdc5/ruby_on_rails/bootstrap.md#L22) first have to be required?

Comment (LionelLindt) [reply ruby_on_rails/bootstrap.md]: image
